cmd/run/socket/proxy.go

```diff
@@ -355,6 +360,9 @@ func (p *proxy) proxyHTTP1(cli, srv *bufConn) error {
 				defer resp.Body.Close()
 				io.Copy(io.Discard, resp.Body)
 			}()
+
+			parser.UseLogs(p.global.Logger, eventID)
```

Comment:
why this the point when we stop the log counter? we've only read the response status code and headers, not the full body. I think the `UseLogs` call should be after the `io.Copy(io.Discard, resp.Body)` call in the above goroutine to ensure that we track all log lines until the response payload is finished.